nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVReader.java

```diff
@@ -54,6 +54,26 @@
         "The first non-comment line of the CSV file is a header line that contains the names of the columns. The schema will be derived by using the "
             + "column names in the header and assuming that all columns are of type String.");
 
+    // CSV parsers
+    public static final AllowableValue APACHE_COMMONS_CSV = new AllowableValue("commons-csv", "Apache Commons CSV",
+            "The CSV parser implementation from the Apache Commons CSV library.");
+
+    public static final AllowableValue JACKSON_CSV = new AllowableValue("jackson-csv", "Jackson CSV",
+            "The CSV parser implementation from the Jackson Dataformats library");
+
+
+    public static final PropertyDescriptor CSV_PARSER = new PropertyDescriptor.Builder()
+            .name("csv-reader-csv-parser")
+            .displayName("CSV Parser")
+            .description("Specifies which parser to use to read CSV records. NOTE: Different parsers may support different subsets of functionality, "
+                    + "and/or exhibit different levels of performance.")
```

Comment:
Suggest changing the NOTE to:

Different parsers may support different subsets of functionality and may also exhibit different levels of performance.